### DISCUSSION TITLE
DT mod in osu!mania should not get higher OD

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#25197] [Ideas] [answered]
Previous Version to score osu!mania, the Score V1, with osu!Stable, when a player applies DT mod, the OD is still the same as before. In detail, MAX judging area remains to be +-16.5/16.33 or so, 300 to be +-40.5ms (OD 8), as written in osu!wiki. But with osu!lazer, when DT mod (1.5x) applied, the judging area of MAX becomes +-11.0ms, as if applied HR and DT at the same time, even more strict than HR only with osu!Stable (MAX=+-11.5ms), which is UNPLAYABLE for players with 60Hz screens, or I have to say SS UNACHIEVABLE, such as MacBook. 

All MAX judgement is also not practical for every player, I do think that everyone have the right to achieve SS on any Beatmap, with 300 judgement, really reasonable. I have mentioned that in [previous discussion](https://github.com/ppy/osu/discussions/23157). SS with ALL MAX and 300, or simply add a Classic Mod for osu!mania.

**Comment by peppy** [comment, marked as answer]
> But with osu!lazer, when DT mod (1.5x) applied, the judging area of MAX becomes +-11.0ms, 

How are you testing this? It was fixed in August (see https://github.com/ppy/osu/pull/24636), so you may be on an old version if you're still experiencing this.

> All MAX judgement is also not practical for every player

This is changing in the next release, see https://github.com/ppy/osu/pull/25111.
